pkg/clustermanager/controller.go

```diff
@@ -45,6 +45,13 @@ const (
 	controllerName                                   = "logicalcluster-manager"
 )
 
+func LCKeyFunc(ns string, name string) string {
```

Comment:
Looks like this is a local helper function so it shouldn't be exported and maybe should be moved to provider.go where it is called.

Reply:
fixed